src/main/java/edu/kis/powp/jobs2d/drivers/TransformationDriver.java

```diff
@@ -0,0 +1,29 @@
+package edu.kis.powp.jobs2d.drivers;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+
+public class TransformationDriver implements Job2dDriver {
```

Comment:
The name of this driver is confusing. It does not make any "Transformation". It looks like a workaround to avoid adding X & Y getters to commands. Adding getters seems more straightforward (KISS rule).